cmd/inspect.go

```diff
@@ -261,8 +260,17 @@ func populateAnnotations(out io.Writer, refs []*ast.AnnotationsRef) error {
 					fmt.Fprintln(out)
 				}
 
+				if len(a.RelatedResources) > 0 {
+					fmt.Fprintln(out, "Related Resources:")
+					l := make([]listEntry, 0, len(a.RelatedResources))
+					for _, res := range a.RelatedResources {
+						l = append(l, listEntry{removeNewLines(res.Ref.String()), res.Description})
+					}
+					printList(out, l, " ")
+					fmt.Fprintln(out)
```

Comment:
Is `Fprintln` needed here?

Reply:
I meant if the `F` version was required, given how only one argument was passed.. but I realized later its a writer being passed (as opposed to a string), so that's probably right then.

Reply:
Yeah exactly. We're printing nothing, followed by a newline, into `out`.